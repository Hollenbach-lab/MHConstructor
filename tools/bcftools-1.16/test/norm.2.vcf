##fileformat=VCFv4.2
##contig=<ID=1,length=249250621>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	10	.	G	.	.	.	.
1	11	.	T	TCT	.	.	.
1	12	.	C	CTC	.	.	.
1	13	.	T	.	.	.	.
1	14	.	B	BGC	.	.	.
