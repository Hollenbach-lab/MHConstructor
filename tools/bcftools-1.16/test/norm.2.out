##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=1,length=249250621>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	10	.	G	.	.	.	.
1	10	.	g	gTC	.	.	.
1	10	.	g	gtC	.	.	.
1	13	.	T	.	.	.	.
1	14	.	a	aGC	.	.	.
